pkgs/build-support/gcc-wrapper/default.nix

```diff
@@ -204,11 +212,11 @@ stdenv.mkDerivation {
       cp -p ${./utils.sh} $out/nix-support/utils.sh
 
       if [ -e $out/bin/clang ]; then
-        echo 'export CC; : ''${CC:=clang}' >> $out/nix-support/setup-hook
+        echo 'export CC=clang' >> $out/nix-support/setup-hook
```

Comment:
Ah yes, the `setup-hook` here sets `CC=gcc` and `CXX=g++`, but if we can just use `cc` and `c++` instead that'd be better :)